app/views/static/middleware/list_providers.html

```diff
@@ -0,0 +1,40 @@
+<div class="row scrollable-sm" id="center_div" style="height: 100%">
+  <div class="col-md-10 col-md-push-2 max-height">
+    <div class="row toolbar-pf">
+      <div class="col-md-12">
+        <div class="toolbar-pf-actions">
+          <miq-toolbar-menu toolbar-items="mwCtrl.toolbarItems"></miq-toolbar-menu>
+        </div>
+      </div>
+    </div>
+    <div class="row" id="main-content" style="">
+      <div class="col-md-12">
+        <div class="row">
+          <div class="col-md-7">
+          <br>
+          <h1>
+            Middleware Managers
+          </h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <miq-data-table columns="mwCtrl.columnsToShow" data="mwCtrl.data" on-row-click="mwCtrl.onRowClick($event, rowData)"
+            on-item-selected="mwCtrl.onRowSelected()" default-action="mwCtrl.defaultAction"></miq-data-table>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="col-md-2 col-md-pull-10 sidebar-pf sidebar-pf-left" style="height: 100vh">
+    <div>
+      <br>
+      <div class="alert alert-info">
+        <span class="pficon pficon-info"></span>
+        <strong>
+          No filters defined.
+        </strong>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
```haml
#center_div.row.scrollable-sm{:style => "height: 100%"}
  .col-md-10.col-md-push-2.max-height
    .row.toolbar-pf
      .col-md-12
        .toolbar-pf-actions
          %miq-toolbar-menu{"toolbar-items" => "mwCtrl.toolbarItems"}
    #main-content.row
      .col-md-12
        .row
          .col-md-7
            %br/
            %h1
              = _("Middleware Managers")
        .row
          .col-md-12
            %miq-data-table{:columns           => "mwCtrl.columnsToShow",
                            :data              => "mwCtrl.data",
                            "default-action"   => "mwCtrl.defaultAction",
                            "on-item-selected" => "mwCtrl.onRowSelected()",
                            "on-row-click"     => "mwCtrl.onRowClick($event, rowData)"}
  .col-md-2.col-md-pull-10.sidebar-pf.sidebar-pf-left{:style => "height: 100vh"}
    %div
      %br/
      .alert.alert-info
        %span.pficon.pficon-info
        %strong
          = _("No filters defined.")
```

.. otherwise, you'd have to do i18n in JS..